controller/src/main/java/com/epam/esm/controller/CustomExceptionHandler.java

```diff
@@ -0,0 +1,115 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.entity.ErrorResponse;
+import com.epam.esm.exception.CertificateNotFoundException;
+import com.epam.esm.exception.IdInvalidException;
+import com.epam.esm.exception.InvalidDataFormException;
+import com.epam.esm.exception.TagNotFoundException;
+import lombok.AllArgsConstructor;
+import org.springframework.context.MessageSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * The type Exception controller.
+ */
+@ControllerAdvice
+@ResponseBody
+@AllArgsConstructor
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
+
+    private final MessageSource messageSource;
+
+    private static final int INTERNAL_SERVER_ERROR_CODE = 100;
+
+    /**
+     * Handles the certificateNotFoundException class
+     *
+     * @param exception the exception
+     * @param locale    the locale
+     * @return the response entity
+     */
+    @ExceptionHandler(CertificateNotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleControllerNotFoundException(CertificateNotFoundException exception, Locale locale) {
+        String errorMessage = messageSource.getMessage("error.certificateNotFound", new Object[]{}, locale);
+        ErrorResponse response = new ErrorResponse();
+        response.setErrorMessage(errorMessage + ' ' + exception.getMessage());
```

Comment:
Please review this class and decide if any relevant constants should be declared.

Reply:
done!